### ISSUE TITLE
striim pay fails when given X number of decimal places

### ISSUE THREAD

**Issue by jacobotoll** [hubiinetwork/striim-cli#20]
`jacobo@$:> (02:25 PM) striim pay 0.0000001 ETH to 4b60a8cd91ea7e463d1e7f238974ad4fed18824f
Error: Payment failed: invalid value`

It works with one less zero.

**Comment by morfj** [comment]
Reproduced locally. Full error callstack:
```
{ Error: invalid value
    at throwError (/home/morten/dev/hubii/striim-cli/node_modules/ethers/utils/throw-error.js:4:17)
    at Object.parseUnits (/home/morten/dev/hubii/striim-cli/node_modules/ethers/utils/units.js:96:9)
    at Object.handler (/home/morten/dev/hubii/striim-cli/src/commands/pay.js:20:41)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3 input: '1e-7' }
Error: Payment failed: invalid value
    at Object.handler (/home/morten/dev/hubii/striim-cli/src/commands/pay.js:36:19)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
```

**Comment by morfj** [comment]
YARGS was parsing positional arguments as numbers and then serializing them back to strings, and in accordance with ECMA specs number smaller than 1e-6 will be serialized in scientific notation. The ethers library however, does not support parsing scientific notation strings :(

**Comment by morfj** [comment]
Fixed in v1.0.8
